packages/wp-story-editor/src/components/metaBoxes/metaBoxesProvider.js

```diff
@@ -44,21 +48,27 @@ function MetaBoxesProvider({ children }) {
   const state = useMemo(
     () => ({
       state: {
+        isSavingMetaBoxes,
         metaBoxesVisible,
         metaBoxes,
         locations,
         hasMetaBoxes,
+        apiUrl,
```

Comment:
Where is that being used? `useSaveMetaboxes` gets it straight from `useConfig`

Reply:
Not being used anywhere, thanks.